Backend/NUnit_Collections/NUnit_Collection_Tests/Collections.cs

```diff
@@ -161,5 +179,6 @@ public void Test_Collection_Clean()
             Assert.AreEqual(expected, actual);
         }
 
```

Comment:
Remove empty spaces.

Reply:
This empty spaces are ok the code has been formatted I don't think that this is a problem. This is just a style. :)